src/Nest/Mapping/DynamicTemplate/SingleMapping.cs

```diff
@@ -34,6 +34,7 @@ public class SingleMappingDescriptor<T> :
 		public IProperty Binary(Func<BinaryPropertyDescriptor<T>, IBinaryProperty> selector) =>
 			selector?.Invoke(new BinaryPropertyDescriptor<T>());
 
+		[Obsolete("Removed in NEST 6.x.")]
```

Comment:
I think we can help users here by pointing them to using the `ingest-attachment` plugin in the comment.

Reply:
Added.